SpatialGDK/Source/SpatialGDK/Public/Schema/UnrealObjectRef.cpp

```diff
@@ -3,6 +3,8 @@
 #include "Schema/UnrealObjectRef.h"
 
 #include "EngineClasses/SpatialPackageMapClient.h"
+#include "SpatialConstants.h"
+#include "EngineClasses/SpatialActorChannel.h"
```

Comment:
We need this header?

Reply:
If so I guess these headers could be reordered.

Reply:
Ah, true - also I think we no longer need SpatialActorChannel, maybe not even SpatialConstants, now that I moved some things into TryResolve...longname.

Reply:
Removed both